packages/aws-cdk-lib/aws-apigatewayv2-integrations/lib/websocket/aws.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  WebSocketRouteIntegration,
+  WebSocketIntegrationType,
+  WebSocketRouteIntegrationConfig,
+  WebSocketRouteIntegrationBindOptions,
+} from '../../../aws-apigatewayv2';
+import { IRole } from '../../../aws-iam';
+
+/**
+ * Props for AWS type integration for an HTTP Api.
+ */
+export interface WebSocketAwsIntegrationProps {
+  /**
+   * Integration URI.
+  */
+  readonly integrationUri: string;
+
+  /**
+   * Specifies the integration's HTTP method type.
+  */
+  readonly integrationMethod: string;
+
+  /**
+   * Specifies the credentials role required for the integration.
+   *
+   * @default - No credential role provided.
+  */
+  readonly credentialsRole?: IRole;
+
+  /**
+   * The request parameters that API Gateway sends with the backend request.
+   * Specify request parameters as key-value pairs (string-to-string
+   * mappings), with a destination as the key and a source as the value.
+   *
+   * @default - No request parameter provided to the integration.
+  */
+  readonly requestParameters?: { [dest: string]: string };
+
+  /**
+   * A map of Apache Velocity templates that are applied on the request
+   * payload.
+   *
+   * ```
+   *   { "application/json": "{ \"statusCode\": 200 }" }
+   * ```
+   *
+   * @default - No request template provided to the integration.
+  */
+  readonly requestTemplates?: { [contentType: string]: string };
+
+  /**
+   * The template selection expression for the integration.
+   *
+   * @default - No template selection expression provided.
+  */
+  readonly templateSelectionExpression?: string;
+}
+
+/**
+ * AWS WebSocket AWS Type Integration
+ */
+export class WebSocketAwsIntegration extends WebSocketRouteIntegration {
+  /**
+   * @param id id of the underlying integration construct
+   */
+  constructor(id: string, private readonly props: WebSocketAwsIntegrationProps) {
+    super(id);
+  }
+
+  bind(options: WebSocketRouteIntegrationBindOptions): WebSocketRouteIntegrationConfig {
+    options;
```

Comment:
this line can't be doing anything at all right? If you don't plan on using `options` you can specify `_` before the prop name i.e. `bind(_options: WebSocketRouteIntegrationBindOptions)`. I think that's what you're trying to do here.

Reply:
Generally that's also my preferred way of dealing with unused parameters. But I seem to find that this is CDK preferred way of dealing with unused parameters as I repetitively see this kind of syntax and that's why I used it. 
```
  bind(options: WebSocketRouteIntegrationBindOptions): WebSocketRouteIntegrationConfig {
    options;
```

Reply:
Where do you see that? Feel free to fix that, it is wrong. Definitely not a CDK preferred way.

Reply:
I've seen some in other files under apigatewayv2. I can fix those in a subsequent PR.